tests/Fixture/ToastMessage.elm

```diff
@@ -1,13 +1,51 @@
-module Fixture.ToastMessage exposing (toastFuzzer)
+module Fixture.ToastMessage exposing (encodeV1, toastFuzzer, v1Fuzzer)
 
 import Fuzz exposing (Fuzzer, int, list, string)
+import Json.Decode as Decode
 import Json.Encode as Encode
+import LabeledMessage exposing (withLabel)
 import Message.Toast exposing (Toast)
 
 
 toastFuzzer : Fuzzer Toast
 toastFuzzer =
-    Fuzz.map3 Toast
+    Fuzz.map4 Toast
+        Fuzz.string
+        Fuzz.string
+        (Fuzz.maybe (Fuzz.constant Encode.null))
+        (Fuzz.oneOf
+            [ Fuzz.constant "info"
+            , Fuzz.constant "warning"
+            , Fuzz.constant "error"
+            ]
+        )
+
+
+
+-- Old Versions
```

Comment:
To test support for old messages, I just copied over the old `Toast` type and renamed it to `ToastV1`, and also moved the old fuzzer and encoder to generate JSON in the old format.